packages/apps/origin-ui/.eslintignore

```diff
@@ -2,6 +2,7 @@ dist/
 node_modules
 assets
 nginx
+cypress
```

Comment:
Why is it ignored?

Reply:
Sorry for late answer, it is ignored because received @typescript-eslint/parser error about unused test files